chp-docker-entrypoint

```diff
@@ -41,4 +41,4 @@ case "$@" in
     ARGS="--api-ip=0.0.0.0";;
 esac
 
-exec configurable-http-proxy $ARGS "$@"
+exec bin/configurable-http-proxy $ARGS "$@"
```

Comment:
## Explanation of line
Due to the change from `npm install -g` to `npm ci` where the `-g` flag isn't available, we need to reference the local executable instead.

I tried copying this to /usr/local/bin but then I got the following error, so I think this is a good solution.

```
/srv/configurable-http-proxy $ ./chp-docker-entrypoint 
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '../package.json'
Require stack:
- /usr/local/bin/configurable-http-proxy
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/usr/local/bin/configurable-http-proxy:11:9)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/local/bin/configurable-http-proxy' ]
}
```

Reply:
Hmmm while it works, because the current working directory is the same as this files location, but I think it is a bit unrobust. So, I'll add the folder to path instead from the Dockerfile.

Reply:
I updated the Dockerfile's PATH env var to find the ./bin folder